pkgs/by-name/li/linux-wallpaperengine/package.nix

```diff
@@ -0,0 +1,75 @@
+{ lib
+, stdenv
+, cmake
+, fetchFromGitHub
+, ffmpeg
+, freeglut
+, freeimage
+, glew
+, glfw
+, glm
+, libGL
+, libpulseaudio
+, libX11
+, libXau
+, libXdmcp
+, libXext
+, libXpm
+, libXrandr
+, libXxf86vm
+, lz4
+, mpv
+, pkg-config
+, SDL2
+, SDL2_mixer
+, zlib
```

Comment:
Should go: lib, stdenv, fetchers then ASCII sorted from there with any Darwin deps after that